R/agg_scale.R

```diff
@@ -28,13 +28,25 @@
 #'   `prod` (for probabilities).
 #' @param missing_dt_severity \[`character(1)`\]\cr
 #'   How severe should the consequences of missing data that prevents
-#'   aggregation or scaling from occurring be? Can be either 'stop', 'warning',
-#'   'message', or 'none'. If not "stop", then only the possible aggregations or
-#'   scaling is done using the available data.
-#' @param drop_present_aggs \[`logical(1)`\]\cr
+#'   aggregation or scaling from occurring be? Can be either 'skip', 'stop',
+#'   'warning', 'message', or 'none'. If 'skip', the checks for missing data are
+#'   skipped. If not 'stop', then only the possible aggregations or scaling is
+#'   done using the available data.
+#' @param present_agg_severity \[`logical(1)`\]\cr
+#'   How severe should the consequences of aggregate data (versus only the most
+#'   detailed level be)? Can be either 'skip', 'stop', 'warning', 'message',
+#'   or 'none'. If 'skip', then the data will be included in any aggregates
+#'   because the checks will be skipped. If anything else other then 'stop', the
+#'   data will be dropped rather than included in possible aggregations.
 #'   Whether to drop aggregates (or overlapping intervals) that are already
 #'   present in `dt` before aggregating. Default is 'False' and the function
 #'   errors out.
+#' @param overlapping_dt_severity \[`character(1)`\]\cr
+#'   When `collapse_interval_cols = TRUE`, how severe should the consequences of
+#'   overlapping intervals that change or prevent collapsing to the most
+#'   detailed common set of intervals be? Can be either 'skip', 'stop',
+#'   'warning', 'message', or 'none'. If not "stop", then overlapping intervals
```

Comment:
Add an explicit note to details section to use `skip` to speed up the function when you know the data is square?